src/vs-threading.MainThreadAssertingMethods.txt

```diff
@@ -1 +1,4 @@
-[SonarLint.VisualStudio.Core.IThreadHandling]::ThrowIfNotOnUIThread
\ No newline at end of file
+# List all of our methods that assert we are on the UI thread.
+# Format: see https://github.com/microsoft/vs-threading/blob/main/doc/analyzers/configuration.md#methods-that-assert-the-main-thread
```

Comment:
Miscellaneous: added some comments to the existing config file to help future maintainers.